packages/kbn-tinymath/README.md

```diff
@@ -65,3 +61,12 @@ parse('1 + random()')
 #### Notes
 
 * Floating point operations have the normal Javascript limitations
+
+### Building kbn-tinymath
+
+This package is rebuilt when running `yarn kbn bootstrap`, but can also be build directly
+using `yarn build` from the `packages/kbn-tinymath` directory.
```

Comment:
I suggest recommending `yarn kbn run build -i @kbn/tinymath`, which is the standard way to build a package from anywhere in the repo.